src/main/java/touro/spf/Graph.java

```diff
@@ -7,15 +7,18 @@
 
 public class Graph {
     private HashSet<Node> nodes = new HashSet<>();
+    public static int numOfGraphs = 0;
 
     public void buildGraph(HashSet<Node> ns) {
         nodes = ns;
+        numOfGraphs++;
```

Comment:
I added this static int variable numOfGraphs to keep track of the number of graphs that are made. This is for when we want to make multiple graphs and want to output what network # the spf found is a part of. Network # is displayed in the example output that Professor showed us that is why I thought it necessary to add. 

This numOfGraphs is incremented arguably in the wrong place. I was thinking of the best place to increment it. Like if we wanted the InputManager could even create multiple instances of graphs and then we would be incrementing this in the constructor. Multiple graph instances may make it easier for the View to handle multiple graphs...

Reply:
ok, i hear you. I would say that the number of graphs should maybe be passed into a constructor though. actually, maybe the `buildGraph` itself should be ripped out, and just happen in the constructor as well.
I dont think you can increment numOfGraphs inside a graph, because every new graph will start again at 0 when it's created, it has to be in a larger context, like main (or whatever)

Reply:
oh wait I take the constructor comment back (in regards to buildgraph) because graph has to be passed to the input manager

Reply:
I kept it as is for now. Let me know if you have a vision for a better place for numOfGraphs with one instance of graphs.